recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0rc1" %}
+{% set version = "1.4.0" %}
 {% set org = "cyclus" %}
-{% set branch = "develop" %}
+{% set branch = "master" %}
 {% set sha256 = "7a6bd1334f3fd13ddac73fbb7a5218b958b793b81bda26552e9c3abbf255a905" %}
```

Comment:
It is the hash of the whole tarball. This value is then checked by conda-build to makes sure that the tarball that conda-build downloads is correct (no download corruption, man-in-the-middle, etc).  The hash algorithm that is recommended by conda/conda-forge is the SHA256 algo. Though we could use this to MD5 or SHA1 if we wanted.

It is not the commit SHA, which is the hash of only that changeset in git.  This hash needs to be generated independently of the version control system. Does this make sense?